.github/workflows/test.yml

```diff
@@ -92,3 +130,18 @@ jobs:
           account_id: ${{ secrets.NEW_RELIC_ACCOUNT_ID }}
           api_key: ${{ secrets.NEW_RELIC_API_KEY }}
           license_key: ${{ secrets.NEW_RELIC_LICENSE_KEY }}
+
+      - name: Run e2e-test
```

Comment:
nit: Maybe add PHP to the name to help with troubleshooting failures?

Reply:
Fixed here: ec2745dd919d0da949f26e47e07295a139362659